lib/kino/explorer.ex

```diff
@@ -37,6 +37,7 @@ defmodule Kino.Explorer do
 
   @impl true
   def init({df, name}) do
+    df = if lazy?(df), do: DataFrame.collect(df), else: df
```

Comment:
you could also call `collect` and it is a no-op for eager data frames. But the issue is that a lazy data frame can potentially be really large and I don't think we want to collect it. Instead, I think we should render it page by page, and some information such as `n_rows` and statics won't be available. And perhaps we change the title to "Lazy DataFrame".